### DISCUSSION TITLE
Synchronise animated gifs

### DISCUSSION THREAD

**Discussion by hellocatfood** [ImageMagick/ImageMagick#2519] [Help]
I've made a script for overlaying multiple gifs onto an image. https://github.com/hellocatfood/gif-sparkle-dust/blob/master/sparkle_sparkle.sh Depending on the duration of the first gif, some gifs either get cut off or end and don't loop.

I noticed that animation synchronisation was mentioned here http://www.imagemagick.org/Usage/anim_mods/#merge

Is there an automated way to do this yet? I'm using ImageMagick 6.9 on Ubuntu 20.04

**Comment by fmw42** [comment]
Nothing new that I know. See https://imagemagick.org/Usage/anim_mods/#merging
